src/DotNet.Status.Web/Models/DeploymentEntity.cs

```diff
@@ -0,0 +1,19 @@
+using Azure;
+using Azure.Data.Tables;
+using System;
+
+namespace DotNet.Status.Web.Models
+{
+    public class DeploymentEntity : ITableEntity
+    {
+        public string PartitionKey { get; set; }
+        public string RowKey { get; set; }
+        public DateTimeOffset? Timestamp { get; set; }
+        public ETag ETag { get; set; }
+
+        public string Service => PartitionKey;
```

Comment:
Do we only ever read these values?  If we ever wrote one down, this would cause this property to create a new row... I think they added some sort of attribute to account for that, somewhere?

Reply:
After I added this, I realized the "most correct" thing for me to do would be to update all other uses of Azure Tables in this project to use the latest Azure SDK library, then merge the model classes (I didn't notice the difference originally, and looking now it's messy to combine the different-yet-too-similar models).

Reply:
I'll look at making a better model here. Or, if you see strong value I can look at updating all the other Table-users to be more aligned.

Reply:
I'm concerned that someone will be like "oh, this model is better, let's use it for the writing too", which will have a weird cascade effect.  The writing will ad the column, and suddenly the reader will start crashing with the serialized data because the proeprty doesn't have a writer anymore. :-)

Reply:
Found the attribute.